docker/run/tomato-ctl.py

```diff
@@ -401,34 +303,46 @@ def read_config():
 				'data': os.path.join("backend_core", "data"),
 				'config': os.path.join("backend_core", "config"),
 				'logs': os.path.join("backend_core", "logs")
-			}
+			},
+			'code_directories': ['backend_core', 'shared'],
+			'shell_cmd': "/bin/bash"
 			# 'version'  (will be generated if not found in config)
 		},
-		'web': {
-			'docker_container': 'tomato_web',
-			'image': 'tomato_web',
-			'port': 8080,
+		'db': {
+			# enabled: True,
+			# is_database: True
```

Comment:
This is automatically added at the end of read_config() (https://github.com/GLab/ToMaTo/pull/1172/files#diff-f8297362b794263e43115ab641e4f11cR389)
The comments are just there to indicate that this will be in the config, like 'version'.